src/components/books/form.js

```diff
@@ -1,26 +1,56 @@
 import './css/form.scss';
-import React from 'react';
+import React, { useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { addBook } from '../../redux/books/books';
 
 function form() {
+  const [title, setTitle] = useState('');
+  const [category, setCategory] = useState('');
+  const length = useSelector(({ booksReducer }) => booksReducer.length);
+  const dispatch = useDispatch();
+
+  const submitBookToStore = (e) => {
+    e.preventDefault();
+    dispatch(
+      addBook({
+        id: `item${length + 1}`,
+        title,
+        category,
+      }),
+    );
+
+    setTitle('');
+    setCategory('');
+  };
+
   return (
     <div className="form-container">
       <h3 className="form-title">ADD NEW BOOK</h3>
-      <div className="form">
-        <input
-          type="text"
-          className="input-title"
-          name=""
-          id=""
-          placeholder="Book title"
-        />
-        <select name="categories" id="category-id" className="input-categories">
-          <option value="Action">Action</option>
-          <option value="Fiction">Fiction</option>
-          <option value="Economy">Economy</option>
-        </select>
-        <button type="button" value="" className="btn-submit">
-          ADD BOOK
-        </button>
+      <div className="forms">
+        <form onSubmit={submitBookToStore} className="form">
+          <input
+            type="text"
+            className="input-title"
+            value={title}
+            placeholder="Book title"
+            onChange={(e) => setTitle(e.target.value)}
+          />
```

Comment:
- [OPTIONAL] I think it'll be good to have an input field for authors too rather than have a default author for all books.

Reply:
👍 Let me fix that. Thank you for the suggestion @ZeenatLawal  😊